Cargo.toml

```diff
@@ -29,3 +29,5 @@ harness = false
 [features]
 default = ["rayon"]
 platform_independent = []
+aarch64_neon = []
```

Comment:
For clarity, this is only a feature because it will depend on a nightly compiler. There's no reason to treat it differently, otherwise. Correct? In this case I would rename the feature to `nightly_aarch64_neon`.

Reply:
Done.